BIBLIOGRAPHY.md

```diff
@@ -101,4 +101,21 @@
 
 [LP] **Linear Programming** http://en.wikipedia.org/wiki/Linear_programming
 
+[ROCHADE] **ROCHADE: Robust Checkerboard Advanced Detection for Camera Calibration**
+    David Fleet and Tomas Pajdla and Bernt Schiele and Tinne Tuytelaars.
```

Comment:
if you are referring to this one https://link.springer.com/chapter/10.1007/978-3-319-10593-2_50 the authors are:
```suggestion
    Simon Placht, Peter Fürsattel, Etienne Assoumou Mengue, Hannes Hofmann, Christian Schaller, Michael Balda and Elli Angelopoulou 
```

Reply:
Thanks, I completely missed that one !